### PR TITLE
Added write CLI access for Extreme switches

### PR DESCRIPTION
**Pull request by dwlfrth**
# Description

Added ability to grant CLI write access for Extreme switches
# Impacts

Extreme switches module
# Delete branch after merge

YES
# NEWS file entries
## New Features
- Added the ability to grant CLI write access for Extreme switches

### CODE REVIEW

**Comment by dwlfrth** [comment]
Followed doc: https://gtacknowledge.extremenetworks.com/articles/How_To/How-to-configure-RADIUS-authentication-for-switch-management-access-in-EXOS

**Comment by julsemaan** [comment]
Requires quick-doc in the network config guide

**Comment by fdurand** [comment]
A doc like that : https://gtacknowledge.extremenetworks.com/articles/How_To/How-to-configure-RADIUS-authentication-via-CLI-on-800-series-switches ?

**Comment by julsemaan** [comment]
Waiting for doc and will review + merge

**Comment by dwlfrth** [comment]
Bump

File: docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc [hunk 1]
```diff
@@ -2319,6 +2320,21 @@ Netlogin (802.1X)
 
 NOTE: You can mix the MAC Authentication and 802.1X on the same switchport. If the device fails 802.1X authentication, it will roll back to the MAC Authentication.
 
+RADIUS authentication for CLI access
+++++++++++++++++++++++++++++++++++++
+
+Configure radius server ipaddress as primary server and the switch's ipaddress as the client-ip.  Be sure to specify the correct virtual router
+
+  configure radius mgmt-access primary server <SERVER_IP> <SERVER_UDP_PORT> client-ip <CLIENT_IP> vr <VR>
```
Comment (fdurand) [inline docs/PacketFence_Network_Devices_Configuration_Guide.asciidoc]: The server port is 1815 for cli access

**Comment by dwlfrth** [comment]
Pushing to 6.5 since need to test something and don't have access to test switch.

**Comment by extrafu** [comment]
Bump.

File: lib/pf/Switch/Extreme.pm [hunk 2]
```diff
@@ -1517,6 +1517,26 @@ sub getVoiceVlan {
...
+    $radius_reply_ref->{'Service-Type'} = 'Administrative-User';
+    $radius_reply_ref->{'Reply-Message'} = "Switch enable access granted by PacketFence";
+    $logger->info("User $args->{'user_name'} logged in $args->{'switch'}{'_id'} with write access");
+    my $filter = pf::access_filter::radius->new;
+    my $rule = $filter->test('returnAuthorizeWrite', $args);
+    ($radius_reply_ref, $status) = $filter->handleAnswerInRule($rule,$args,$radius_reply_ref);
+    return [$status, %$radius_reply_ref];
+}
+
 =back
```
Comment (fdurand) [inline lib/pf/Switch/Extreme.pm:1560]: Missing returnAuthorizeRead.

For read:
Service-Type = 0

**Comment by extrafu** [comment]
@dwlfrth Make sure it does work and merge.

**Comment by dwlfrth** [comment]
Review and merge

**Comment by dwlfrth** [comment]
Review me and merge

**Pull request merged**